flexmeasures/data/scripts/visualize_data_model.py

```diff
@@ -228,6 +246,11 @@ def show_image(graph, fb_viewer_command: str):
         help="Postgres user (needed if --schema is on).",
         default="flexmeasures",
     )
+    parser.add_argument(
```

Comment:
I recommend including this as a separate feature in the changelog. It's worth a note in our docs, too.

Reply:
You mean the entire script is worth a note in the docs? Because I don't think we mention it at all yet.

Actually, all the commands from whole `Makefile` are worth a page in the developer docs ...